.travis.yml

```diff
@@ -23,9 +20,7 @@ env:
 install:
   - pip install -U pip wheel
   - pip install -U -r requirements/dev.txt
-  - nvm install 10.16.0
-  - nvm use 10.16
-  - npm install
+  - pip install -U -r requirements/test.txt
```

Comment:
Since test.in includes dev.in, we don't need to install the requirements from both dev.txt and test.txt. test.txt should include them all.

Reply:
O I understand. I'll make changes here and in `circle.yaml`

Reply:
done